project1/css/index.css

```diff
@@ -62,3 +62,103 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+a{
+	color:gray;
+	text-decoration:none;
+}
+ul{
+	position:relative;
+	top:8px;
+}
+li {
+	display:inline;
+	margin:0 50px 0 50px;
+}
+ul{
+	margin:0 0 80px 0;
+}
+button{
+	background-color:white;
+	border-radius:0px;
+	padding:10px 40px 10px 40px;
+	border-style:solid;
+	margin:20px 0px 0px 80px;
+}
+footer p{
+	margin-top:1300px;
+	margin-left:400px;
+}
+.logo{
+	position:absolute;
+	float:right;
+	top:0px;
+	margin:0 0 0 920px;
+}
+.cta img {
+	position:relative;
+	float:right;
+}
+.cta-text{
+	float:left;
+	margin: 0 0 0 200px;
+}
+.top-content .text-content p{
+	width:300px;
+	position:absolute;
+	margin: 0 200px 0 0;
+	height:100px;
+	
+}
+
```

Comment:
Inconsistent spacing between different blocks of code. All of the code up to this point had no space between blocks. 

Consistency is key!